### DISCUSSION TITLE
Функционал и контроль админов.

### DISCUSSION THREAD

**Discussion by Yus228** [schr1k/AnonChat#4] [General]
Привет, подскажи, какой функционал у админов? Как выявлять нарушителей и как их банить? За что отвечает команда /admin и \n, ничего не происходит, когда их ввожу. Айди админов указал. И еще вопрос, Фото/видео не сохраняются у меня на пк?

**Comment by schr1k** [comment]
Привет, команда /admin работает только внутри диалога, а точнее, насколько я помню вообще не работает). Команды /n нету, функционала у админов как такового нет. Фото, видео и остальной тип контента не сохраняется локально, они отправляются через id в чате.

**Comment by schr1k** [comment]
Я немного забросил данный проект, но сейчас понемногу возвращаюсь. Обновления сюда наверное закидывать не буду, хотя если звездочек 10 наберётся, то могу.

**Comment by Yus228** [comment]
Жди звездочек, а я жду контента. А если не секрет, куда ты еще выкладываешь контент?

**Comment by schr1k** [comment]
В закрытый репозиторий, сейчас переписываю клавиатуры на инлайн и пытаюсь делать проверку оплаты через вебхуки, но к сожалению у payok ужасная документация(исходный код тоже смотрел и там ни намека на вебхуки).

**Reply by ArtKorshun** [reply]
Не рассматривал использовать киви? оплата через ipa токен весьма удобна +комиссии нет

**Reply by schr1k** [reply]
![image](https://github.com/schr1k/AnonChat/assets/110760070/50d2d908-279c-43b9-881f-a9b2b27c000e)
Рассматривал)
